### ISSUE TITLE
Terminating server from environment fails; Leaves orphaned instance on server

### ISSUE THREAD

**Issue by doublerebel** [Distelli/Support#4]
Hello,

At apps/livebassmusicrightnow?section=overview I can see both entries of my app listed under Instances on the Right.  However I attempted to Terminate this app from nebula in my environment lbmrn-prod and it failed ( deployments/d-59udfc0vrk7f8b73a670 ).  How do I terminate this running instance of my app?

This happened once before and I reported it to Brian using the web chat support.  Unfortunately I was in a hurry and had already deleted the files from my server and restarted the distelli Agent in order to force the fix, and that left no evidence.  This time I have not deleted anything or restarted the agent in hopes of solving this bug.  (I do not today see any web chat support or any other support available in help or from the dashboard.)

Thanks,

Charles

**Comment by doct15** [comment]
Hi.
The web chat support channel is the best place to get support.
Your link references do not provide enough information for me to look into the issue.
Happy to help! As noted, best to reach us on the web chat support channel.
Thanks!

**Comment by doct15** [comment]
Hi. I was able to hunt down your account info from your other issue.
Sorry you ran into this. I just took a look. The workaround is:
1. navigate to env lbmrn-prod
2. Add a server
3. Re-add 10.2.0.15 (do not allow auto-deploy of active release)
4. Save
5. Remove a server
6. Remove 10.2.0.15
7. Yes, terminate app from server.

I hope I was not to forward in executing those steps and terminating that app from that server for you.
Here is the termination task:
/deployments/d-m9usdnct0w44f0b45422

Interesting, if you compare that termination task to your earlier one that failed, the "Environment" differs. Not sure under what conditions may have caused this, but that is why the server did not recognize the app being previously deployed.

**Comment by doublerebel** [comment]
I appreciate you looking into this and terminating the process successfully, not too forward at all.  Thanks for confirming the removal process for me.  I thought I had tried that sequence of actions, so I'll make sure to follow your steps next time.

It looks like Privacy Badger started to block the Intercom widget, I whitelisted the widget URLs and I see the chat bubble again.

There is definitely something weird on that server, I deployed another branch ( nextorigin/deployments/d-ponsmktmb37da54a6b8 ) and it doesn't seem to log to anywhere.  There is no folder for it in /distelli/envs and the stdout streaming to the Distelli dashboard isn't working, but I can see it in the process list and it's registered itself as healthy in my Consul service catalog.  I'm starting to wonder if that particular agent installation went bad somehow, I rebooted the server and the new deployment is still without logs.  I tried to examine it with pfiles to find an open log file but did not see anything.

This is odd but workable for now, I'm sure I'll just toss it and create another instance before long.  If this continues into a new cloud instance then I'll open a ticket in the web chat.

Thanks again for your help.
